compiler/damlc/daml-lf-conversion/src/DA/Daml/LFConversion/Primitives.hs

```diff
@@ -280,10 +280,9 @@ convertPrim _ "UFetch" (TContractId (TCon template) :-> TUpdate (TCon template')
     ETmLam (mkVar "this", TContractId (TCon template)) $
     EUpdate $ UFetch template (EVar (mkVar "this"))
 
-convertPrim version "USoftFetch" ty@(TContractId (TCon template) :-> TUpdate (TCon template'))
+convertPrim _ "USoftFetch" (TContractId (TCon template) :-> TUpdate (TCon template'))
```

Comment:
I think you can remove this too since we never generate `primitive @"USoftFetch"`

Reply:
same goes for `USoftExercise`

Reply:
Thanks! As you mentioned on slack these were dead code already, I'll address that in a separate PR.

Reply:
Done in https://github.com/digital-asset/daml/pull/18440